Merge original README

ferdymercury opened the pull request:
Fixes https://github.com/martukas/nuclei/issues/7

ferdymercury committed:
Merge original README

Fixes https://github.com/martukas/nuclei/issues/7

martukas commented on README.md:
```diff
@@ -2,14 +2,21 @@
 
+## Features
+
+Nuclei is a software tool for the displaying of nuclear decay schemes and estimated energy spectra, the calculation of angular γ emission anisotropies, and the automated search for appropriate decay cascade properties.
```
I am uncertain if that's still true. In my modernization, I think I removed the sprectra generation feature.

martukas commented on README.md:
````diff
@@ -39,6 +46,11 @@ conan remote add conancenter https://center.conan.io
 
+If on Ubuntu 22, install:
+```bash
+sudo apt install libqt5svg5-dev
````
I might even simplify this `Building` section with simply a pointer to the circleci script to see what packages are installed. What bulids on CI is the closest thing to a guarantee of functionality. Everything else is hard to state with any certainty and/or maintain.

martukas commented on README.md:
```diff
@@ -62,12 +74,21 @@ Parsing errors will be printed in terminal as the files are loaded.
+M. Nagl, et al., NIM A 726 (2013), 17-30, [doi:10.1016/j.nima.2013.05.045](https://doi.org/10.1016/j.nima.2013.05.045)
+
+-------------------------
+© 2012-2013, Matthias Nagl, II. Physikalisches Institut, Georg-August-Universität Göttingen
```
Probably the old license no longer applies? Also, the specific link is dead and just goes to the institute's main page. Maybe we don't need this snippet?

martukas reviewed (changes requested):
Thanks! Sorry it took a while to respond to this.
I have provided a few suggestions.

ferdymercury committed:
amend for changes in current version, simplify build instructions in readme, point instead at circleCI

suggestions by martukas

martukas reviewed (approved):
LGTM

(other than CI failing)

ferdymercury commented:
@php1ic it seems that the circleci does not run well. Could it maybe be a multithreading issue?

php1ic commented:
Yeah, I have seen this before on circleci, sometimes the build fails if you let things build without specifying how many cores/threads to use.

https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-build-j

Simplest way is to add an argument after `--parallel` on [line 27](https://github.com/martukas/nuclei/blob/main/.circleci/config.yml#L27) of the config file. Looks like it uses a 2 core CPU so that is the number we should use.

ferdymercury committed:
fix CI

due to a problem in not finding out well the number of jobs

sugessted by php1ic
